libuuu/buffer.cpp

```diff
@@ -1186,6 +1186,17 @@ shared_ptr<FileBuffer> get_file_buffer(string filename, bool async)
 	}
 }
 
+void *uuu_get_file_buffer(const char *in_filename, size_t *out_size)
+{
+	shared_ptr<FileBuffer> fb = get_file_buffer(in_filename);
+	if (fb == NULL) {
+		return NULL;
+	}
+	else {
+		*out_size = fb->size();
+		return (void*)fb->data();
+	}
+}
 
```

Comment:
Sorry, I found a problem with export uuu_get_file_buffer.  shared_ptr fb will auto decrease reference number when function return. 
C++ have not provide a simple way to increase reference number.  

So fb->data() may already free after this function return.